app/Post.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Post extends Model
+{
+    //
+    public $timestamps = false;
```

Comment:
Typically you would leave the timestamps because its always useful to know when stuff happened. Laravel manages it out of the box.

Reply:
I would normally leave this in, however in the testing phase when doing
`$this->assertDatabaseHas('posts', $post->toArray());`
I was experiencing errors as the newly created `Post` object had time stamps in a different format/parsed differently to those in the database already. See https://github.com/laravel/framework/issues/31920. What would be the solution you would use?

Reply:
I'm not sure what I have done differently, however testing works now using the exact same line `$this->assertDatabaseHas('posts', $post->toArray());` when I uncomment the timestamps line during migration.

Reply:
I would probably do 
```
$this->assertDatabaseHas('posts', [
  'name' => 'expected',
  'description' => 'hello',
]);
```
rather than using toArray(). You should be able to say 
```
'created_at' => $post->created_at
```
although that is not a particularly useful assertion because you are just testing a framework feature.